TotalCrossVM/CMakeLists.txt

```diff
@@ -0,0 +1,620 @@
+cmake_minimum_required(VERSION 3.13.5)
+
+# set the project name
+project(TCVM VERSION 1.0)
+
+if(NOT DEFINED BUILD_SHARED_LIBS AND NOT DEFINED TCVM_SHARED)
+  if(CMAKE_GENERATOR STREQUAL Xcode)
+    set(TCVM_SHARED OFF) 
+    message("tcvm: ${TCVM_SHARED}")  
+  else() # others
+    set(TCVM_SHARED ON) 
+  endif()
+endif()
+
+# Find SDL2 and Skia
+set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/modules")
+
+# Find SDL2
+IF (NOT (DEFINED ANDROID_ABI OR MSVC OR CMAKE_GENERATOR STREQUAL Xcode))
+  find_package(SDL2 REQUIRED)
+  include_directories(${SDL2_INCLUDE_DIR})
+ENDIF (NOT (DEFINED ANDROID_ABI OR MSVC OR CMAKE_GENERATOR STREQUAL Xcode))
+
+# Find Skia
+if(NOT MSVC)
+  find_package(Skia REQUIRED)
+  include_directories(${SKIA_INCLUDE_DIRS})
+endif(NOT MSVC)
+
+# Set C/CXX flags
```

Comment:
This block fails here, so I think it should be dropped.

Reply:
Can you give more details about the failure?

Reply:
fixed